apps/sequencer_tests/bin/sequencer_tests.rs

```diff
@@ -548,6 +549,49 @@ async fn main() -> Result<()> {
         assert_eq!(expected_response, actual_response);
     }
 
+    fn mask_timestamps(response: &str) -> String {
```

Comment:
Interesting technique, seems like I like it. Today I learned something new .. yeah !

Reply:
Thanks, it's a bit of a hack, but I thought it's good enough for this test :-)